pkgs/by-name/ma/manifold/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  clipper2,
+  gtest,
+  glm,
+  tbb,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "manifold";
+  version = "2.5.1-unstable-2024-08-16";
+
+  src = fetchFromGitHub {
+    owner = "elalish";
+    repo = "manifold";
+    rev = "d4fb82d4d1e83a861064253c68d6fb0a980499e0";
+    hash = "sha256-SDmNzWdRmykMqPUMkeBHx3mMUN3YZbYnedqiN4DmocU=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [
+    gtest
+    glm
+    tbb
+    clipper2
+  ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DBUILD_SHARED_LIBS=ON"
+    "-DMANIFOLD_TEST=ON"
```

Comment:
I think `MANIFOLD_TEST` is default to `ON` already?

Reply:
I wanted it to be explicit, as `doCheck = true;` is set. This makes it independent of the defaults in the CMakeLists.